src/Components/CreateImageWizardV2/ImageWizardContext.tsx

```diff
@@ -0,0 +1,208 @@
+import React, { useState } from 'react';
+
+import {
+  EnvironmentStateType,
+  filterEnvironment,
+  useGetAllowedTargets,
+} from './steps/ImageOutput/Environment';
+import { GCPAccountTypes } from './steps/TargetEnvironment/GCP/GCPTarget';
+import { useGetAccountData } from './steps/TargetEnvironment/SourcesSelect';
+
+import { RHEL_9, X86_64 } from '../../constants';
+import { ArchitectureItem, Distributions } from '../../store/imageBuilderApi';
+
+type ImageWizardContextPropType = {
+  releaseState: [
+    Distributions,
+    React.Dispatch<React.SetStateAction<Distributions>>
+  ];
+  architectureState: [
+    ArchitectureItem['arch'],
+    React.Dispatch<React.SetStateAction<ArchitectureItem['arch']>>
+  ];
+  environmentState: [
+    EnvironmentStateType,
+    React.Dispatch<React.SetStateAction<EnvironmentStateType>>
+  ];
+  isAwsManualState: [boolean, React.Dispatch<React.SetStateAction<boolean>>];
+  awsSourceState: [
+    [number, string],
+    React.Dispatch<React.SetStateAction<[number, string]>>
+  ];
+  associatedAwsAccountIdState: [
+    string,
+    React.Dispatch<React.SetStateAction<string>>
+  ];
+  isShareGoogleAccountState: [
+    boolean,
+    React.Dispatch<React.SetStateAction<boolean>>
+  ];
+  gcpAccountTypeState: [
+    GCPAccountTypes,
+    React.Dispatch<React.SetStateAction<GCPAccountTypes>>
+  ];
+  gcpAccountEmailState: [string, React.Dispatch<React.SetStateAction<string>>];
+  gcpDomainState: [string, React.Dispatch<React.SetStateAction<string>>];
+  isAzureManualState: [boolean, React.Dispatch<React.SetStateAction<boolean>>];
+  azureSourceState: [
+    [number, string],
+    React.Dispatch<React.SetStateAction<[number, string]>>
+  ];
+  azureTenantIdState: [string, React.Dispatch<React.SetStateAction<string>>];
+  azureSubscriptionIdState: [
+    string,
+    React.Dispatch<React.SetStateAction<string>>
+  ];
+  azureResourceGroupState: [
+    string,
+    React.Dispatch<React.SetStateAction<string>>
+  ];
+};
+
+export const ImageWizardContext =
+  React.createContext<ImageWizardContextPropType>({
+    releaseState: [RHEL_9, () => {}],
+    architectureState: [X86_64, () => {}],
+    environmentState: [
```

Comment:
We usually call aws/gcp/azure/etc... the 'target', don't we? Could we change 'environment' to 'target'?

Reply:
Oh I'm dumb, it is the 'target environment'. Still, it confused me so maybe there is some inconsistency. Could we call it the `targetEnvironment` (dropping the `-State` suffix)?

Reply:
environment was the naming used in the previous wizard. If you prefer targetEnvironment I'm all for it.